easycukes-scm/src/main/java/com/worldline/easycukes/scm/stepdefs/SCMStepdefs.java

```diff
@@ -45,6 +46,21 @@
 public class SCMStepdefs {
 
     protected static final EasyCukesConfiguration<CommonConfigurationBean> configuration = new EasyCukesConfiguration<>(CommonConfigurationBean.class);
+    
+	/*
+	 *  Used to convert base url into scm base url 
+	 */
+	private static final String SCM_BASE_URL = "/portal";

```

Comment:
Not at all!

EasyCukes is a generic framework, it shouldn't contain anything related to any of our internal tools.